diff -urN linux-headers.orig/arch/riscv/include/uapi/asm/setup.h linux-headers/arch/riscv/include/uapi/asm/setup.h
--- linux-headers.orig/arch/riscv/include/uapi/asm/setup.h	1970-01-01 02:00:00.000000000 +0200
+++ linux-headers/arch/riscv/include/uapi/asm/setup.h	2017-06-13 08:46:15.567800383 +0300
@@ -0,0 +1,6 @@
+/*
+ * setup.h is part of userspace header ABI so UAPI scripts have to generate it
+ * even if there's nothing to export - causing empty <uapi/asm/setup.h>
+ * However to prevent "patch" from discarding it we add this placeholder
+ * comment
+ */
